dask_sql/physical/rex/core/call.py

```diff
@@ -133,7 +133,13 @@ def reduce(self, *operands, **kwargs):
                 )
             ):
                 operands = tuple(map(as_timelike, operands))
-            return reduce(partial(self.operation, **kwargs), operands)
+            try:
+                return reduce(partial(self.operation, **kwargs), operands)
+            except ValueError:
+                return reduce(
+                    partial(self.operation, **kwargs),
+                    (operands[0], float(operands[1][operands[1].columns[0]].loc[0].compute()))
```

Comment:
I'd imagine we'd want to generalize the typecast here to handle other potential aggregating functions, but ATM can't think of an immediate way to do this.